delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/utils/ClasspathResourceUtils.scala

```diff
@@ -0,0 +1,80 @@
+package ch.epfl.bluebrain.nexus.delta.kernel.utils
+
+import scala.io.{Codec, Source}
+import io.circe.Json
+import io.circe.parser.parse
+import org.fusesource.scalate.TemplateEngine
+
+// $COVERAGE-OFF$
+trait ClasspathResourceUtils {
```

Comment:
We should probably test these, to immediately identify when we have problems.
Additionally, since the code is used in "compile" scope and it's side effecting, I strongly recommend changing the signatures to return `Task[A]`; this would also allow for memoization.